shared/checks/oval/installed_OS_is_rhel8.xml

```diff
@@ -41,14 +41,16 @@
     <linux:name>redhat-release</linux:name>
   </linux:rpminfo_object>
 
-  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="redhat-release-coreos is version 8" id="test_rhel8_coreos" version="1">
-    <linux:object object_ref="obj_rhel8_coreos" />
-    <linux:state state_ref="state_rhel8_coreos" />
-  </linux:rpminfo_test>
-  <linux:rpminfo_state id="state_rhel8_coreos" version="1">
-    <linux:version operation="pattern match">^[0-9\.]*8.*$</linux:version>
-  </linux:rpminfo_state>
-  <linux:rpminfo_object id="obj_rhel8_coreos" version="1">
-    <linux:name>redhat-release-coreos</linux:name>
-  </linux:rpminfo_object>
+  <ind:textfilecontent54_test check="all" comment="redhat-release-coreos is version 8" id="test_rhel8_coreos" version="1">
+    <ind:object object_ref="obj_rhel8_coreos" />
+    <ind:state state_ref="state_rhel8_coreos" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_rhel8_coreos" version="1">
+    <ind:filepath>/etc/os-release</ind:filepath>
+    <ind:pattern operation="pattern match">^PRETTY_NAME=&quot;Red Hat Enterprise Linux CoreOS \d+\.(\d+)\.\d+\.\d+ \([\w\s]+\)&quot;$</ind:pattern>
```

Comment:
@ashcrow what do you think of this, to check if a node is RHCOS?

Reply:
`NAME` would be a simpler check if all that matters is if it's RHCOS or not ... but I think this would work. NOTE: I just eyeballed the regex :slightly_smiling_face:

Reply:
I hope that it did not put too much strain to your eyes. I'm open to suggestions for improvements of the regex used in the check :)